YUViewLib/src/playlistitem/playlistItemRawFile.cpp

```diff
@@ -558,8 +564,22 @@ void playlistItemRawFile::getSupportedFileExtensions(QStringList &allExtensions,
   allExtensions.append("nv21");
   allExtensions.append("y4m");
 
+  allExtensions.append("rgba");
+  allExtensions.append("rbga");
+  allExtensions.append("grba");
+  allExtensions.append("gbra");
+  allExtensions.append("brga");
+  allExtensions.append("bgra");
+  allExtensions.append("argb");
+  allExtensions.append("arbg");
+  allExtensions.append("agrb");
+  allExtensions.append("agbr");
+  allExtensions.append("abrg");
+  allExtensions.append("abgr");
```

Comment:
Do you think these are missing? This was not metioned in the issue.

Reply:
I will open another issue, since I found YUView supports RGB image with alpha channel.
With these appending, opening *.rgba files will be more convenient.